pagefaultgames/pokerogue
# [Mirror][Localization] Translate forgotten phrases sentences #2254

## What are the changes?
These changes are a mirror of https://github.com/pagefaultgames/pokerogue/pull/2254

## Why am I doing these changes?
The owner of the previous PR @Nolway seems to have stopped working on it.
To continue previous work, I made a new PR and branch on EnochG1/mirror-translate-forgot-phrases

## What did change?
1. Move owners of the [translate-forgot-phrases](https://github.com/Nolway/pokerogue/tree/translate-forgot-phrases) branch
2. Resolve conflict with latest 'main' branch.
3. Remove duplicate message ("You received {{modifierName}}!" is already localized at https://github.com/pagefaultgames/pokerogue/pull/2556)

### Screenshots/Videos
#### En
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/0b0e6585-f70f-4e80-8cc7-ea2f34007da9) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/edd7f27c-da2e-4568-863c-25a28ff27dc8) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/5fdcf57e-5ed2-4fe8-af01-67a50413202e) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/b6e5b022-09fc-4085-ae8e-762f6324036e) |

</details>

#### Fr
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/6dc2d12b-9fef-474a-b9ba-81a0e70e08aa) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/2b77fb24-0e68-4741-aa38-b7a8540b5552) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/7d73169f-4364-4c6d-b192-5d63100b6df4) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/9015e55b-cbaa-48ef-ab42-6bf729c8405a) |

</details>

#### De
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/47e8bc42-f9ab-4bb5-a933-6a8da6f44884) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/6340c603-dd8b-4f76-bfc7-568fde6c635d) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/84d877c9-49dc-419f-9dbb-06593a2844fc) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/953f16b2-848e-4282-a18f-fef56b7eaeee) |

</details>

#### Pt_BR
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/973dad55-9da7-4fce-8d40-641e81acaaf0) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/b537d167-99bf-42a3-a2c8-a7d53eb28831) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/aa56c70f-2f06-4855-8d6b-823727197fdf) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/345cf2be-8ac9-4ac5-af16-b5b06244dc3d) |

</details>

#### Zh_CN
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/31d66cac-c08c-420d-b4e6-4a5cb96281e6) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/886c2406-3630-42c0-a1b9-14727c91e455) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/3d4e1c4c-4031-4d08-b995-34bc2461f7d3) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/9fd12860-75e8-4f2a-a655-455fd9aca785) |

</details>

#### Ko
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/6a5377c0-e798-4df1-87b0-a6476c610f08) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/5cc410be-1720-421a-9b73-2e66cf5b0ba7) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/62dfed6a-9cda-42a3-9451-998c5f69e16e) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/1785c940-a68c-4cb0-a606-90674ea85764) |

</details>

#### It
<details>
<summary>Click to expand/collapse</summary>

| Case | Result |
| :-----: | :-----: |
| retryBattle | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/57101ed6-edaf-4476-84be-515032f8f4d5) |
| congratulations | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/6a913ef7-d2df-4ab5-9f34-7e46d014104d) |
| unlockedSomething | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/875e3300-804f-452c-aeb9-d0f071672a45) |
| beatModeFirstTime | ![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/30791dbd-f86b-446b-93da-269e02578c76) |

</details>


## How to test the changes?
Manually.
1. retryBattle : Make all pokémon fainted then check message.
2. unlockedSomething / congratulations / beatModeFirstTime : Clear Classic Mode with pokémon that have no history of completing Classic Mode. Using override.ts like this.
```Typescript
export const STARTING_WAVE_OVERRIDE: integer = 200;
export const STARTING_BIOME_OVERRIDE: Biome = Biome.END;
export const STARTER_FORM_OVERRIDES: Partial<Record<Species, number>> = {
  [Species.VENUSAUR]: 2
export const STARTING_LEVEL_OVERRIDE: integer = 500;
export const STARTER_SPECIES_OVERRIDE: Species | integer = Species.VENUSAUR;
};
```

## Checklist
 - Only with https://github.com/pagefaultgames/pokerogue/pull/2254 which was mirrored
